cypress_test/integration_tests/desktop/writer/page_style_dialog_spec.js

```diff
@@ -0,0 +1,25 @@
+/* global describe it cy beforeEach require */
+
+var helper = require('../../common/helper');
+
+describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Page Style dialog tests', function() {
+	beforeEach(function() {
+		helper.setupAndLoadDocument('writer/styles.odt');
+		cy.viewport(1920, 720);
+	});
+
+	/*
+	 * tests if the width of the 'Page Style dialog' is larger than a "reasonable"
+	 * width and if it's larger that means something is obviously wrong, probably
+	 * some css property.
+	 * `reasonableWidth` = width at the time of writing this test + 15px ;)
+	 */
+	it('Page Style dialog width', function() {
+		cy.cGet('#Format-tab-label').click();
+		cy.cGet('#format-page-dialog176-button').should('exist').should('be.visible');
+		cy.cGet('#format-page-dialog176-button').click();
+
+		cy.cGet('#TemplateDialog8').should('exist').should('be.visible');
```

Comment:
please join the two lines, only `be.visible` is needed, it has to exist to be visible :)